python/ray/util/sgd/v2/backends/horovod.py

```diff
@@ -0,0 +1,89 @@
+import logging
+import os
+from dataclasses import dataclass
+from typing import Optional, Set
+
+import ray
+from horovod.ray.runner import Coordinator
```

Comment:
Can you make the horovod imports optional, like in `TorchConfig` and `TFConfig`?